src/components/Section/ProposalGovernanceSection.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useMemo } from 'react'
+
+import Time from 'decentraland-gatsby/dist/utils/date/Time'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+
+import { ProposalAttributes, ProposalStatus, ProposalType } from '../../entities/Proposal/types'
+import { SelectedVoteChoice, Vote } from '../../entities/Votes/types'
+import { ProposalPageState } from '../../pages/proposal'
+import { ChoiceProgressProps } from '../Status/ChoiceProgress'
+
+import ProposalVotingSection from './ProposalVoting/ProposalVotingSection'
+
+import './DetailsSection.css'
+import { ProposalPromotionSection } from './ProposalPromotionSection'
+import ProposalThresholdsSummary from './ProposalThresholdsSummary'
+
+type ProposalGovernanceSectionProps = {
+  proposal?: ProposalAttributes | null
+  votes?: Record<string, Vote> | null
+  partialResults: ChoiceProgressProps[]
+  choices: string[]
+  loading?: boolean
+  disabled?: boolean
+  onChangeVote?: (e: React.MouseEvent<unknown>, changing: boolean) => void
+  onVote: (selectedChoice: SelectedVoteChoice) => void
+  castingVote: boolean
+  proposalPageState: ProposalPageState
+  updatePageState: (newState: Partial<ProposalPageState>) => void
+  handleScrollTo: () => void
+}
+
+const PROMOTABLE_PROPOSALS = [ProposalType.Poll, ProposalType.Draft]
+
+export default function ProposalGovernanceSection({
+  proposal,
+  loading,
+  disabled,
+  votes,
+  partialResults,
+  choices,
+  onChangeVote,
+  onVote,
+  castingVote,
+  proposalPageState,
+  updatePageState,
+  handleScrollTo,
+}: ProposalGovernanceSectionProps) {
+  const now = Time.utc()
+  const finishAt = Time.utc(proposal?.finish_at)
+  const finished = finishAt.isBefore(now)
+  const showPromotionSection = useMemo(
+    () => proposal && proposal.status === ProposalStatus.Passed && PROMOTABLE_PROPOSALS.includes(proposal.type),
+    [proposal]
+  )
+  const showProposalThresholdsSummary = !!(
+    proposal &&
+    proposal?.required_to_pass != null &&
+    proposal?.required_to_pass >= 0 &&
+    !(proposal.status === ProposalStatus.Passed)
+  )
+
+  if (!showPromotionSection && !showProposalThresholdsSummary && !!finished) return null
```

Comment:
Same as before

Reply:
For this particular case I'd rather leave them here. 
Its not a simple check, and each of the individual variables are used to render child components